src/components/cta/cta.tsx

```diff
@@ -24,7 +24,44 @@ export class CtaComponent {
   // optional message prop
   @Prop() public message?: string;
 
+  /**
+   * Set href attribute if you want the output to be link else button will be created
+   */
+  @Prop() public href?: string;
```

Comment:
We will need more properties, and appropriate handling for those. 

Like: Target, Id, rel, aria, class, data, eventAction, eventLabel, eventCategory, assistiveText, host, tracking, IconName, IconText.